enderio-base/src/main/java/com/enderio/base/common/item/tool/PoweredToggledItem.java

```diff
@@ -112,4 +118,27 @@ public boolean shouldCauseReequipAnimation(ItemStack oldStack, ItemStack newStac
         return oldStack.getItem() != newStack.getItem() || Toggled.isEnabled(oldStack) != Toggled.isEnabled(newStack);
     }
 
+    @Override
+    public boolean isBarVisible(ItemStack pStack) {
+        return true;
+    }
+
+    @Override
+    public int getBarWidth(ItemStack pStack) {
+        return pStack
+            .getCapability(CapabilityEnergy.ENERGY)
+            .map(energyStorage -> Math.round((float)energyStorage.getEnergyStored() * 13.0F / (float)energyStorage.getMaxEnergyStored()))
+            .orElse(0);
+    }
+
+    @Override
+    public int getBarColor(ItemStack pStack) {
+        return ItemBarRenderer.ENERGY_BAR_RGB;
+    }
```

Comment:
Would you mind explaining to me, how the frick both the energy bar as well as the durability bar are rendered?
To my knowledge are these methods used to render the durability and you render both, even though you override these methods?

Reply:
hmm, come to think of it now I'm confused too 😆 I'll do some digging and try make some comments if necessary. This was just a very quick port pass haha

Reply:
PoweredToggledItem only affects the electromagnet and staff of levitation. This has nothing to do with the dark steel tools, which use a custom render overlay.